components/common/Main.tsx

```diff
@@ -29,8 +30,16 @@ const Main = ({ children }: MainProps): JSX.Element => {
             justifyContent="center"
             minHeight="100vh"
         >
-            {isLarge && <LeftNav />}
-            <Box>{user ? children : <PleaseLogInPage isLarge={isLarge} />}</Box>
+            {isLarge && <LeftNav {...auth} />}
+            <Box>
+                {user && user.email ? (
```

Comment:
I think we can just assume that if a user managed to log in they also have an email